core/zap/module.go

```diff
@@ -80,6 +82,13 @@ func (m *Module) Inject(config *struct {
 	m.logSession = config.LogSession
 	m.callerEncoder = callerEncoders[ZapCallerEncoderShort]
 
+	//if not provided set 1 as default
+	m.callerskip = 1
```

Comment:
yes, as you mentioned, let's set the default value to 2

```suggestion
	m.callerskip = 2
```